### PR TITLE
fix: remove disabling on send input click

### PR DESCRIPTION
**Pull request by alter-eggo**
> Try out this version of the Hiro Wallet - download [extension builds](https://github.com/hirosystems/wallet/actions/runs/5198025639).

Tackled all the comments from https://github.com/hirosystems/wallet/issues/3136 except showing available under fees list, which is done in this pr https://github.com/hirosystems/wallet/pull/3801

### CODE REVIEW

File: src/app/pages/send/send-crypto-asset-form/family/bitcoin/components/bitcoin-send-max-button.tsx [hunk 1]
```diff
@@ -58,7 +58,7 @@ export function BitcoinSendMaxButton({
         width="fit-content"
         {...props}
       >
-        Sending max
+        {isSendingMax ? 'Sending max' : 'Send max'}
```
Comment (kyranjamie) [inline src/app/pages/send/send-crypto-asset-form/family/bitcoin/components/bitcoin-send-max-button.tsx:61]: Wasn't this logic already there in Fara's PR? 🤔

**Review by kyranjamie — APPROVED** [review]

File: src/app/pages/send/send-crypto-asset-form/family/bitcoin/components/bitcoin-send-max-button.tsx (hunk 1 above)
Comment (alter-eggo) [reply src/app/pages/send/send-crypto-asset-form/family/bitcoin/components/bitcoin-send-max-button.tsx:61]: seems not https://github.com/hirosystems/wallet/pull/3782/files#diff-8e1fb4e649a6d0e4554384b65f954c97dbc2c98794b12c3a46aae7526e9085a7

**Comment by mica000** [comment]
@alter-eggo not sure what is happening. Clicking on send max givers a error and then clicking again, enables the sending max?

https://github.com/hirosystems/wallet/assets/119523541/007c7847-f46f-4cec-b303-d61f96469db1

**Comment by alter-eggo** [comment]
@mica000 your exact problem is fixed here https://github.com/hirosystems/wallet/pull/3820

but there is other one I mentioned in the comments

**Comment by mica000** [comment]
> @mica000 your exact problem is fixed here #3820
> 
> but there is other one I mentioned in the comments

not sure I fully comprehend what are you referring to?

**Comment by alter-eggo** [comment]
> Clicking on send max givers a error and then clicking again, enables the sending max?

this is fixed in new pr https://github.com/hirosystems/wallet/pull/3820

**Pull request merged**